scripts/check_doc_ipfs_hashes.py

```diff
@@ -32,7 +32,7 @@
 
 
 CLI_REGEX = r"(?P<cli>aea)"
-CMD_REGEX = r"(?P<cmd>.*)"
+CMD_REGEX = r"(?P<cmd>(?!-)\S+(\s--\S+)*)"
```

Comment:
This PR comes with no test that lets us assert the correct behaviour. Please add one.